exchange/seat_non_bids.go

```diff
@@ -40,6 +41,10 @@ func (snb *nonBids) addBid(bid *entities.PbsOrtbBid, nonBidReason int, seat stri
 	snb.seatNonBidsMap[seat] = append(snb.seatNonBidsMap[seat], nonBid)
 }
 
+// get returns a slice of SeatNonBid objects representing the non-bids for each seat.
+// If snb is nil, it returns nil.
+// It iterates over the seatNonBidsMap and appends each seat and its corresponding non-bids to the seatNonBid slice.
+// Finally, it returns the seatNonBid slice.
 func (snb *nonBids) get() []openrtb_ext.SeatNonBid {
```

Comment:
Consider changing the struct to a type alias to keep things simple and remove a level of indirection and checks:

```
type SeatNonBidBuilder map[string][]openrtb_ext.NonBid
```

and renaming this function to make its transform clearer with a normal receiver instead of a pointer receiver, something like:

```
func (b SeatNonBidBuilder) Slice() []openrtb_ext.SeatNonBid {
 ...
}
```

this more closely follows the pattern of BytesBuffer.

Reply:
@SyntaxNode: This is already implemented thing. Do you want it to be refactored?